react-ui/src/__snapshots__/App.test.js.snap

```diff
@@ -0,0 +1,119 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
snapshot code can be effectively ignored as they are created automatically and used only for testing comparisons.